packages/webapp/src/components/CustomSignUp/index.js

```diff
@@ -0,0 +1,61 @@
+import styles from './styles.scss';
+import Logo from '../../assets/images/signUp/logo.svg';
+import LineBreak from '../../assets/images/signUp/lineBreak.svg';
+import Button from '../Form/Button';
+import Input from '../Form/Input';
+import React from 'react';
+import Footer from '../Footer';
+import GoogleLoginButton from '../../containers/GoogleLoginButton';
+
+export default function PureCustomSignUp({ inputs = [{}], ssoSignUp, onSubmit, disabled }) {
+  var color = '';
+  disabled ? (color = '#D4DAE3') : (color = '#FCE38D');
```

Comment:
Instead of setting the background colour of the button explicitly, it is perhaps better to use the `disabled` props that the `Button` component already has.

Reply:
Might also have the benefit of making the button not clickable before a proper email is inputted

Reply:
Done.